packages/jsActions/nanoflow-actions-native/src/external/DownloadFileByURL.ts

```diff
@@ -0,0 +1,42 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+import fetchBlob from "rn-fetch-blob";
+
+// BEGIN EXTRA CODE
+function formatPath(fileURL: string, filePath: string, fileNameOverride: string): string {
+    const fileName = fileNameOverride || fileURL.substring(fileURL.lastIndexOf("/") + 1);
+    return [filePath, fileName].filter(str => !!str).join("/");
+}
```

Comment:
For consistency, I would move the fileName logic out of this function (for example to line 30 in the main function) and use the exact same implementation as the other action.

Reply:
✨